index.php

```diff
@@ -132,15 +140,47 @@
             $url = sprintf('team/%s', $platform);
             break;
     }
+    return $verifyUrl($url) ? $routes[$url] : $routes['default'];
```

Comment:
I was trying to think how to do this but couldn't remember what it was called, Thanks!